CMSgov/onemac
# OY2 17898: Upgrade Project Node v14

Story: https://qmacbis.atlassian.net/browse/OY2-17898
Endpoint: https://d10z7fv4bmyelq.cloudfront.net

### Details
With the upgrade to Serverless v3, should also upgrade node as far as possible.
AWS Lambdas support up to node 14

### Changes
- updated all configuration files (.yml) to use the node 14 runtime
- fixed .sechub serverless v3 issues
- fixed a problem with sinkMsk's use of Status values
- removed @g-s-k as a CODEOWNER for review purposes

### Test Plan
1. Verify repo deploys correctly
2. Regression test application
3. Use AWS console to view logs for [stream-functions-node-14-sinkMskToDynamo](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/stream-functions-node-14-sinkMskToDynamo) 
4. Verify that it is handling events from Kafka without crashing.
